staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/unstructured/unstructured.go

```diff
@@ -405,7 +427,11 @@ func (u *Unstructured) SetLabels(labels map[string]string) {
 }
 
 func (u *Unstructured) GetAnnotations() map[string]string {
-	m, _, _ := NestedStringMap(u.Object, "metadata", "annotations")
+	m, _, err := NestedStringMap(u.Object, "metadata", "annotations")
+	if err != nil {
+		utilruntime.HandleError(err)
```

Comment:
don't change this accessor to call `utilruntime.HandleError`, that has the potential to fill server logs with lots of inactionable logs